.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: ruby
```

Comment:
thats it ? :D I've never set up travis :P I thought that it's more complicable :D